changes/unreleased/Fixed-20240808-142106.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Fixed
```

Comment:
I thought I had already added this.

Reply:
I didn't see the changie in the PR 835 and the release of 2.1.3.